impl/http/src/main/java/io/serverlessworkflow/impl/executors/http/HttpExecutor.java

```diff
@@ -187,16 +187,17 @@ private static RequestSupplier buildRequestSupplier(
     switch (method.toUpperCase()) {
       case HttpMethod.POST:
         WorkflowFilter bodyFilter = WorkflowUtils.buildWorkflowFilter(application, body);
-        return (request, w, context, node) ->
-            converter.toModel(
-                application.modelFactory(),
-                node,
-                request.post(
-                    converter.toEntity(bodyFilter.apply(w, context, node)), node.objectClass()));
+        return (request, w, context, node) -> {
+          WorkflowModel model = bodyFilter.apply(w, context, node);
+          return converter.toModel(
+              application.modelFactory(),
+              node,
+              request.post(converter.toEntity(model), model.objectClass()));
+        };
       case HttpMethod.GET:
       default:
         return (request, w, t, n) ->
-            converter.toModel(application.modelFactory(), n, request.get(n.objectClass()));
+            converter.toModel(application.modelFactory(), n, request.get(Object.class));
```

Comment:
This should be handled in a more smart way, because although this will work, its doing a lot of Map-JsonNode unneeded conversion. 
Basically converter should be configurable so we can hardcode JsonNode.class there (n.objectClass is not the current response class but the model class, thats the BUG!!!!)